know/rdf/parser/abstract_parser.py

```diff
@@ -0,0 +1,33 @@
+
+from abc import ABC, abstractmethod
+from enum import Enum
+class EntityType(Enum):
+    """ The enum that holds entity types for easy access """
+    #Add more types later
+    PERSON = 1
+    WORK_OF_ART = 2
```

Comment:
Work of art is going to break down into book and painting, so I'd probably remote it